src/Datadog.Trace.ClrProfiler.Native/logger.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "environment_variables.h"
+#include "string.h"
+#include "logger_impl.h"
+
+#include <string>
+
+
+namespace trace
+{
+
+struct TracerLoggerPolicy
+{
+    inline static const std::string file_name = "dotnet-tracer-native";
+#ifdef _WIN32
+    // this field will be removed once merged with the profiler in order to have
+    // the same product folder name
+    inline static const WSTRING product_folder_name = WStr("Datadog .NET Tracer");
+#endif
+    inline static const std::string pattern = "%D %I:%M:%S.%e %p [%P|%t] [%l] %v";
+    struct logging_environment
+    {
+        // cannot reuse environment::log_path variable. On alpine, test fails
+        inline static const WSTRING log_path = WStr("DD_TRACE_LOG_PATH");
```

Comment:
I do not know why on Alpine the test `Datadog.Trace.ClrProfiler.IntegrationTests.NoMultiLoaderTests.SingleLoaderTest` if we have this line instead:
`inline static const WSTRING log_path = environment::log_path;`